config/routes.rb

```diff
@@ -1,3 +1,19 @@
 Rails.application.routes.draw do
+  # get '/invite', to: 'invitations/new'
+  get 'invitations/show'
+  get '/invite', to: 'invitations#new'
+  get 'events/show'
```

Comment:
This route is already covered in `resources :events`. If you only want certain actions you can limit them by inserting them inside an array as specified in the previous comment.

[`REQUIRED`]